homeassistant/components/konnected/sensor.py

```diff
@@ -0,0 +1,124 @@
+"""
+Support for DHT and DS18B20 sensors attached to a Konnected device.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.konnected/
```

Comment:
We're not adding this url anymore to the docstring. Just keep the very first line.